calico/getting-started/kubernetes/managed-public-cloud/eks.mdx

```diff
@@ -38,6 +38,7 @@ When using the Amazon VPC CNI plugin, $[prodname] does not support enforcement o
 ***Prerequisites***
 
 * You [disabled network policy for the AWS VPC CNI](https://docs.aws.amazon.com/eks/latest/userguide/network-policy-disable.html). 
```

Comment:
Hmm, actually, how can this be a pre-req when you've not created your cluster yet!